Source/DUnitX.Utils.pas

```diff
@@ -1121,6 +1121,58 @@ function ConvStr2Enum(const ASource: TValue; ATarget: PTypeInfo; out AResult: TV
   Result := True;
 end;
 
+function Str2DateTime(const ASource: String): TValue;
+var
+  dateTime: TDateTime;
+  iso8601: TFormatSettings;
+begin
+  if(not TryStrToDateTime(ASource, dateTime)) then
+    begin
+      iso8601 := TFormatSettings.Invariant();
```

Comment:
This fails in XE7 - which version was Invariant added in - need to find a way to get this that works from D2010

Reply:
Invariant was added in XE8 - I have pushed a change in master that adds a record helper for TFormatSettings in DUnitX.Utils that should resolve this, please update your pr branch from master and push again.